packages/pigeon/example/app/android/app/src/main/kotlin/dev/flutter/pigeon_example_app/MainActivity.kt

```diff
@@ -64,6 +64,8 @@ class EventListener : StreamEventsStreamHandler() {
     eventSink = sink
   }
 
+  override fun onCancel(p0: Any?) {}
```

Comment:
@stuartmorgan-g @tarrinneal pigeon_example_app failed to build without this change. Does that mean this is a breaking change? If so, does that change what solution you prefer here?

Reply:
Sorry, looking again at the top-level interface class it has default implementations. I really thought it was just abstract declarations.

So your previous version of the PR was the behavior-preserving one, and I was just confused. Sorry about that!